server/src/setup/http/express/basic-app.ts

```diff
@@ -0,0 +1,23 @@
+import { RequestContext } from '@mikro-orm/core'
+import express, { Router } from 'express'
+import { MikroORM } from '../../db'
+import { App } from './app'
+
+export interface BasicAppOptions {
+  mikroORM?: MikroORM
+}
+
+export const setupBasicApp = (expressRouter: Router, options: BasicAppOptions): App => {
+  const app = express()
+  app.use(express.json())
+
+  const entityManager = options?.mikroORM?.em
+  if (entityManager !== undefined) {
+    app.use((_req, _res, next) => RequestContext.create(entityManager, next))
+  }
```

Comment:
Can we move forwards with app setup if `entityManager === undefined` is true?

```suggestion
  } else {
    throw()?
  }
```

Reply:
For the sake of testing, I added mock repos in this PR. In the last commit where I added this code, there is a new `setupMockDB` function. It creates a "Mock DB" which has no ORM.

This means that it is now possible to start up Waterpark using the mock repos instead of the Mikro repos. The `RequestContext` is only needed if Mikro is used.